moped-editor/src/views/staff/StaffForm.js

```diff
@@ -32,7 +32,7 @@ import {
 } from "@material-ui/core";
 import clsx from "clsx";
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
```

Comment:
Lots of Prettier formatting updates in this file. The important updates are around `handleUserActivation` and `disabled` props.